docs/rfcs/XeGPU.md

```diff
@@ -701,6 +700,332 @@ An example on how to perform transpose using load with chunk_size in SIMT flavor
 
 ```
 
+## Workgroup level XeGPU Operations
+
+By allowing XeGPU operating on workgroup level data size, it provides a concise IR for tensor compiler instead of multiple level nested loop IR for subgroup and work item level operation. To enable XeGPU operate the workgroup level, we introduce `wg_map` attribute to specify how the data is distributed across subgroups. `wg_map` enables tensor compiler to express the cooperative operation among subgroups by specifying a `wg_map` to partition data among subgroups without modifying the IR representation other required when using loop nest IR. The attribute allows tensor compiler to control the block size for both the workgroup and subgroup and perform autotuning as the number of subgroups, layout, and tensor size per subgroups are critical performance knobs.
+
+**Attribute xegpu.wg_map**
+
+`wg_map` specifies how a n-d tensor (defined by the tensor descriptor) is partitioned among subgroup within a workgroup. wg_map consists of three parameters:
+  * sg_layout: Defines the n-d arrangement of subgroups within the workgroup. The dimensions can be a 3d array as [dim_0, dim_1, dim_2]. 
+  * sg_data: Specifies the shape of the tensor size for each subgroup after decomposition.
+  * sg_order: The dimension order used to linearize n-d subgroup ids to 1-d. The first dimension in the sg_order list is the fastest-changing dimension. 
+
+Example of linerized subgourp id regarding order[1, 0] vs. order [0, 1]. 
+| sg_layout[4, 4] | order[1, 0] | order[0, 1]
+| :----   | :----   |  :----   |
+| [0, 0], [0, 1], [0, 2], [0, 3] | 0, 1, 2, 3 | 0, 4, 8, 12 |
+| [1, 0], [1, 1], [1, 2], [1, 3] | 4, 5 , 6, 7| 1, 5, 9, 13 |
+| [2, 0], [2, 1], [2, 2], [2, 3] | 8, 9, 10 , 11 | 2, 6, 10, 14 |
+| [3, 0], [3, 1], [3, 2], [3, 3] | 12, 13, 14, 15 | 3, 7, 11, 15 |
+
+For a subgroup threads in 3-d sg_layout [dim_0, dim_1, dim_2], sg_order[2, 1, 0] maps a subgroup thread with 3-d index [x, y, z] to a linear subgroup thread index [z + dim_2*y + dim_2*dim_1*x ], sg_order[1, 2, 0] maps to [y + dim_2*z + dim_2*dim_1*x].
```

Comment:
nit: dim_2*y should be dim_2\*y to show the `*` in markdown.

Reply:
thanks